context/bootstrapper.go

```diff
@@ -63,3 +63,37 @@ func (m *MainBootstrapper) Bootstrap(context map[string]interface{}) error {
 	}
 	return nil
 }
+
+// TenantBootstrapper ensures all dependencies for a tenant is initialised correctly
+type TenantBootstrapper struct {
+	IOCContext    map[string]interface{}
+	Bootstrappers []bootstrap.Bootstrapper
+}
+
+func (t *TenantBootstrapper) Populate() {
```

Comment:
Why don't keep them static?

Reply:
since we might need to change the list dynamically and call bootstrap. Also it follows the same style as main bootstrapper just to keep things consistent. We can refactor later if needed.